users/api/logging.go

```diff
@@ -67,3 +67,16 @@ func (lm *loggingMiddleware) Identify(key string) (id string, err error) {
 
 	return lm.svc.Identify(key)
 }
+
+func (lm *loggingMiddleware) UserInfo(ctx context.Context, key string) (u users.UserInfo, err error) {
+	defer func(begin time.Time) {
+		message := fmt.Sprintf("Method get user info for client %s took %s to complete", u.Email, time.Since(begin))
```

Comment:
Change `get user` part to `user_info`. We always use the method name here with underscore notation.